src/mpo/mpo.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+# Define the Pauli matrices
+sigma_x = np.array([[0, 1], [1, 0]])
+sigma_y = np.array([[0, -1j], [1j, 0]])
+sigma_z = np.array([[1, 0], [0, -1]])
+
+def hamiltonian_to_mpo(hamiltonian, m):
```

Comment:
Let's rename this `convert_hamiltonian_to_mpo`

Reply:
in general, try not to use one letter variables like m - rename to something that describes what this actually is like `number_of_sites`